examples/restaurant_visits/run_without_frameworks_utility_analysis.py

```diff
@@ -112,10 +112,14 @@ def per_partition_utility_analysis():
     data_extractors = get_data_extractors()
     public_partitions = list(range(1, 8)) if FLAGS.public_partitions else None
 
-    result = utility_analysis_engine.aggregate(
-        restaurant_visits_rows, aggregate_params, data_extractors,
-        public_partitions, get_multi_params(),
-        FLAGS.partition_sampling_probability)
+    options = utility_analysis_new.UtilityAnalysisOptions(
+        1,
```

Comment:
Is there a guidance to add comments for non-obvious parameter names in Python? 
In Java this would look like /* epsilon= */ 1. This helps readability.

Reply:
Python has better solution, named arguments, i.e. (epsilon=1, delta = ...)
I've updated the code.